src/tests/class_tests/smartpeak/source/ApplicationProcessor_test.cpp

```diff
@@ -246,4 +247,59 @@ BOOST_AUTO_TEST_CASE(SetOutputFeaturesPathnameGetName)
   BOOST_CHECK_EQUAL(application_processor.getName(), "SetOutputFeaturesPathname");
 }
 
+BOOST_AUTO_TEST_CASE(StoreSequenceWorkflow1)
+{
+  namespace fs = boost::filesystem;
+  ApplicationHandler application_handler;
+  std::vector<std::string> command_names = {
+    "LOAD_RAW_DATA",
+    "MAP_CHROMATOGRAMS",
+    "EXTRACT_CHROMATOGRAM_WINDOWS",
+    "ZERO_CHROMATOGRAM_BASELINE",
+    "PICK_MRM_FEATURES",
+    "QUANTIFY_FEATURES",
+    "CHECK_FEATURES",
+    "SELECT_FEATURES",
+    "STORE_FEATURES"
+  };
+  application_handler.sequenceHandler_.setWorkflow(command_names);
+  StoreSequenceWorkflow application_processor(application_handler);
+  BOOST_CHECK_EQUAL(application_processor.getName(), "StoreSequenceWorkflow");
+  application_processor.pathname_ = (fs::temp_directory_path().append(fs::unique_path().string())).string();
```

Comment:
Is boost filesystem needed here or can the test be accomplished using only the `SMARTPEAK_GET_TEST_DATA_PATH` macro?  This is quite minor so please address at your discretion.

This comment also applies to the similar test below.